deployment/deploy/DeployTask.cpp

```diff
@@ -363,7 +363,7 @@ class CDeployTask : public CInterface, implements IDeployTask
            {
              transform.setResultTarget(target);
              Owned<IFile> pTargetFile = createIFile(target);
-             if (pTargetFile->exists() && pTargetFile->isReadOnly())
+             if (pTargetFile->exists() && pTargetFile->isReadOnly()==fileBool::foundYes)
```

Comment:
The exists test is redundant, and isReadOnly is a poor name.  (test|check)ReadOnly() would be a bit clearer.  is should only be used for boolean functions.